core/templates/dev/head/expressions/ExpressionSyntaxTreeService.ts

```diff
@@ -135,23 +134,17 @@ oppia.factory('ExpressionSyntaxTreeService', [
      * can be system or user defined functions and parameters, as well as
      * system operators.
      * @param {string} name The name to look up.
-     * @param {!Array.<!Object>} envs Represents a nested name space
+     * @param {!Array.<!object>} envs Represents a nested name space
      *     environment to look up the name in. The first element is looked up
      *     first (i.e. has higher precedence).
      * @throws {ExprUndefinedVarError} The named variable was not found in
      *     the given environment.
      */
-    var lookupEnvs = function(name, envs) {
-      // Parameter value look up.
-      var value;
-      if (envs.some(function(env) {
+    const lookupEnvs = (name: string, envs: object[]) => {
```

Comment:
@ankita240796, do we use arrow functions (as `const`) as of now? (Also, do we have eslint for es6 syntax?)

Reply:
I went through gitter and realized we are going to have arrow function in the future. (marking this "resolved")